tools/distrib/check_nanopb_output.sh

```diff
@@ -15,7 +15,6 @@
 
 set -ex
```

Comment:
Do we still need this check?

Reply:
This will be gone once all nanopb code is migrated to upb.